kitsune/sumo/tests/test_parser.py

```diff
@@ -532,19 +532,18 @@ def test_alt_unsafe(self):
         unsafe_vals = (
             (
                 'an"<script>alert()</script>',
-                "an&quot;&amp;amp;lt;script&amp;amp;gt;alert()&amp;amp;lt;/script&amp;amp;gt;",
+                "an&quot;&amp;lt;script&amp;gt;alert()&amp;lt;/script&amp;gt;",
             ),
             (
                 "an'<script>alert()</script>",
-                "an'&lt;script&gt;alert()&lt;/script&gt;",
+                "an'&amp;lt;script&amp;gt;alert()&amp;lt;/script&amp;gt;",
```

Comment:
Both of these test changes were due to an improvement made in the latest version of `bleach`. Our old version of `bleach` provided secure, escaped results, but the latest version provides more accurately escaped results for display purposes.